CHANGELOG.md

```diff
@@ -1,7 +1,26 @@
 # Changelog
 
+## [1.1.9] - 2021-05-31
 
-## [1.1.1] - WIP
+### Added
+
+- Test for tor, connections manager, and dataserver
+- Secured tor control port with hashed password
+- Linter
+- method to close 
+
+### Changed
+
+- Refactored tor control port and tor manager.
```

Comment:
Add information that api changed addNewService -> createNewHiddenService

Reply:
Btw. if API changes we should be bumping major version. We didn't do it but maybe it's a good time to start?